### PR TITLE
Queues Natalia Kuleniuk 

### PR DESCRIPTION
**Pull request by natalia-ku**
# API Muncher
Congratulations! You're submitting your assignment!

## Comprehension Questions
| Question | Answer    |
| :------------- | :------------- |
| How did you go about exploring the Edamam API, how did you try querying the API? | First, I read all documentation to understand what's going on where I created API query . Before creating methods in RecipeApiWrapper class I tested all queries that may help me  in Postman, it helped me a lot to figure out what queries I need and how they should look |
| Describe your API Wrapper.  How did you decide on the methods you created? |  My Api Wrapper has two main functions: find one recipe and find all recipes. Also , I have  a few additional private methods to refactor my main methods' functionality |
| Describe an edge case or failure case test you wrote for your API Wrapper. | Raise error if it cannot find recipe,  fail to search or create new object if no arguments are given |
| Explain how VCR aids in testing an API. | It helped me reduce number of API calls by recording same API calls that I already have used in my tests |
| What is the Heroku URL of your deployed application? | https://nataliaapimuncher.herokuapp.com/  |
| Do you have any recommendations on how we could improve this project for the next cohort? |  |

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+              <button class="close-button small" aria-label="Dismiss alert" type="button" data-close>
+                <span aria-hidden="true">&times;</span>
+              </button>
+          </div>
+            <% end %>
+          </div>
+    </header>
+      <%= yield %>
+  </body>
+  <footer>
```
Comment (droberts-sea) [inline app/views/layouts/application.html.erb:38]: The `<footer>` should still be inside the `<body>`

**Comment by droberts-sea** [comment]
# API Muncher
## What We're Looking For

| Feature | Feedback    |
| :------------- | :------------- |
| **Core Requirements** |   |
| Git hygiene | yes |
| Comprehension questions	| yes |
| **General** |  |
| Rails fundamentals (RESTful routing, use of named paths) | yes |
| Semantic HTML | yes |
| Errors are reported to the user | yes |
| API Wrapper to handle the API requests | yes |
| Controller testing | mostly - what if the user attempts to show a recipe with a bad ID? |
| **Styling** |  |
| Foundation Styling for responsive layout | yes |
| Search View | yes |
| List View | yes |
| Show View which opens in a new tab | yes |
| **API Features** | |
| The App attributes Edaman | yes |
| The VCR casettes do not contain the API key | yes |
|  **Overall** |  |

Great work on the optionals!
